pl_bolts/callbacks/vision/confused_logit.py

```diff
@@ -66,9 +69,10 @@ def __init__(
         self.logging_batch_interval = logging_batch_interval
         self.min_logit_value = min_logit_value
 
-    def on_train_batch_end(self, trainer, pl_module, outputs, batch, batch_idx, dataloader_idx):
+    def on_train_batch_end(self, trainer: Trainer, pl_module: LightningModule, outputs: Sequence,
+                           batch: Sequence, batch_idx: int, dataloader_idx: int) -> None:
```

Comment:
```suggestion
    def on_train_batch_end(
            self,
            trainer: Trainer,
            pl_module: LightningModule,
            outputs: Sequence,
            batch: Sequence,
            batch_idx: int,
            dataloader_idx: int
    ) -> None:
```

@Borda Shall we introduce a code formatter? [Optuna](https://github.com/optuna/optuna) uses [`black`](https://github.com/psf/black) to keep the codebase consistent in formatting, and it worked great without any issues when I previously made contributions there.